### PR TITLE
feat: view all clearpool protocols pools in yields tab

### PR DESCRIPTION
**Pull request by Addis-Belete**
- dynamic
- vault
- treasury

### CODE REVIEW

**Comment by Addis-Belete** [comment]
Log: Some test cases are failing. I'm working on fixing them.

**Comment by Addis-Belete** [comment]
Screenshot 2024-05-21 at 9 44 33 AM

File: src/adaptors/clearpool/index.js [hunk 1]
```diff
@@ -2,70 +2,37 @@
...
-  const dataPools = await utils.getData('https://clearpool.finance/api/pools');
-  let pools = [];
 
-  for (const chainId in dataPools) {
-    for (const pool of dataPools[chainId]) {
-      const chainName = utils.formatChain(networks[chainId]);
-
-      pools.push({
-        pool: `${pool.address}-${chainName}`.toLowerCase(),
-        chain: chainName,
```
Comment (georgeciubotaru) [inline src/adaptors/clearpool/index.js:48]: no sure here is we will preserve the same format as it was, please check

**Review by georgeciubotaru — COMMENTED** [review]
see my comment

File: src/adaptors/clearpool/index.js (hunk 1 above)
Comment (Addis-Belete) [reply src/adaptors/clearpool/index.js:48]: @georgeciubotaru its same format.  I didn't used `poolAddress` & `protocol` fields from the API. all good from my end.
below are the sample pools generated after running the test cases.
- the only difference is in the existing we used both `USDT` and `USDC` as underlying tokens. but now we only used the pool asset.
Screenshot 2024-05-21 at 9 59 35 AM

Comment (georgeciubotaru) [reply src/adaptors/clearpool/index.js:48]: @Addis-Belete I mean the `chain` name

Comment (georgeciubotaru) [reply src/adaptors/clearpool/index.js:48]: https://github.com/clearpool-finance/yield-server/blob/master/src/adaptors/utils.js

File: src/adaptors/utils.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ exports.formatChain = (chain) => {
   )
     return 'zkSync Era';
   if (chain && chain.toLowerCase() === 'polygon_zkevm') return 'Polygon zkEVM';
-  return chain.charAt(0).toUpperCase() + chain.slice(1);
+  return chain.charAt(0).toUpperCase() + chain.slice(1).toLowerCase();
```
Comment (Addis-Belete) [inline src/adaptors/utils.js]: Fixed the function to return the chain name in sentence case

**Comment by Addis-Belete** [comment]
@georgeciubotaru Review now

File: src/adaptors/utils.js (hunk 2 above)
Comment (georgeciubotaru) [reply src/adaptors/utils.js]: @Addis-Belete We shouldn't update these functions, only our folder

File: src/adaptors/clearpool/index.js [hunk 3]
```diff
@@ -2,70 +2,60 @@
...
+          break;
+        case 'ZKEVM':
+          chainName = 'Polygon zkEVM';
+          break;
+        default:
+          chainName = toSentenceCase(pool.chain);
+      }
+      pool.chain = chainName;
+      pools.push(pool);
+    });
```
Comment (georgeciubotaru) [inline src/adaptors/clearpool/index.js]: how about map?

**Review by georgeciubotaru — APPROVED** [review]

**Pull request merged**